### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by proyectoramirez**
This is part of the initial GitHub setup guide

### CODE REVIEW

**Review by Enriqe — APPROVED** [review]
Ship it! 🛳

File: portfolio/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (Enriqe) [inline portfolio/pom.xml]: nit-pick: please delete this comment

**Review by Enriqe — CHANGES_REQUESTED** [review]
Let's delete that comment before moving on

File: portfolio/pom.xml (hunk 1 above)
Comment (proyectoramirez) [reply portfolio/pom.xml]: Done!

**Review by Enriqe — APPROVED** [review]
LGTM

**Pull request merged**